Sources/PointFree/Account/Account.swift

```diff
@@ -340,17 +411,17 @@ private let logoutView = View<Prelude.Unit> { _ in
 }
 
 private let subscriptionInfoRowClass =
-  Class.border.bottom
+  Class.border.top
     | Class.pf.colors.border.gray800
-    | Class.padding([.mobile: [.topBottom: 3]])
+    | Class.padding([.mobile: [.top: 2, .bottom: 3]])
 
-private let labelClass =
+let labelClass =
```

Comment:
we could now moves these into the base styles too now that we are using it more.

I have a simple idea to slightly organize that better if you wanna chat about it today